### PR TITLE
KCE-46-eventloop example

### PR DESCRIPTION
**Pull request by paulagb8894**

### CODE REVIEW

File: tutorials/chapter1.md [hunk 1]
```diff
@@ -350,6 +350,61 @@ With this code, we have ensured that any vector published into the eventloop wil
 
 The last argument, a clone function that the publisher will use instead of standard copy, is also to increase performance. This is covered in depth in the [vision tutorial](https://github.com/klepsydra-technologies/kpsr-vision-ocv-tutorial).
 
+### Benchmark
+
+The next benchmark measures and shows the eventloop performance:
```
Comment (mharshe) [inline tutorials/chapter1.md:355]: We have provided a small benchmarking example to demonstrate the performance benefits of eventloop. The main benchmarking block is shown below, and the complete benchmark is in file `example4c.cpp`

File: examples/chapter1/CMakeLists.txt [hunk 2]
```diff
@@ -83,6 +83,17 @@ TARGET_INCLUDE_DIRECTORIES(${PROJ_NAME}_4b
   ${CMAKE_CURRENT_SOURCE_DIR}/include 
   )
 
+ADD_EXECUTABLE(${PROJ_NAME}_4c ${${PROJ_NAME}_HEADERS} src/example4c.cpp )
```
Comment (mharshe) [inline examples/chapter1/CMakeLists.txt:86]: maybe rename the cpp and target to be `eventloop_benchmark_example`[.cpp]

File: tutorials/chapter1.md (hunk 1 above)
Comment (paulagb8894) [reply tutorials/chapter1.md]: Done

File: examples/chapter1/CMakeLists.txt (hunk 2 above)
Comment (paulagb8894) [reply examples/chapter1/CMakeLists.txt]: Done

**Review by mharshe — APPROVED** [review]

**Pull request merged**